### ISSUE TITLE
ofBook Translation: Japanese

### ISSUE THREAD

**Issue by bakercp** [openframeworks/ofBook#185]
Hi all,

This issue is a place to discuss and organize a Japanese translation. If you are interested or are already working on a Japanese translation, please respond in below.

ping @laserpilot

**Comment by bakercp** [comment]
From @laserpilot 

... but just adding a note here that someone has done a translation of the installationup4evr chapter into Japanese: https://github.com/hideyukisaito/Installation_Up_4evr - if that is another language to be tackled eventually @hideyukisaito

**Comment by tpltnt** [comment]
The following chapters are pretty stable and a good start for the translation efforts:
- of_philosophy
- cplusplus_basics
- setup_and_project_structure
- intro_to_graphics
- OOPs!
- animation

**Comment by tado** [comment]
@bakercp @tpltnt Thanks! I'm interested in translating ofBook into Japanese. I will do it with oF community people in Japan!

**Comment by hideyukisaito** [comment]
@bakercp My translation of the Installation_Up_4evr is almost done, so I'll be happy to merge it into ofBook.

**Comment by arturoc** [comment]
@hideyukisaito can you send a pul request? 

@tado perhaps you can coordinate the japanese translation? i'll give you permissions in this repo so you can review and merge pull requests for japanese.

**Comment by tado** [comment]
@arturoc @hideyukisaito Thanks!

I' will check @hideyukisaito 's pull request, and marge it. And I will coordinate Japanese translations from this.

**Comment by tado** [comment]
@hideyukisaito 
ありがとうございます! リポジトリにPull requestを送ってみていただけますか? それを確認して問題なさそうだったら、Margeします。よろしくお願いします!

**Comment by hideyukisaito** [comment]
@tado 了解です、"almost done" なのでもう少々お待ちを。。

**Comment by kynd** [comment]
Hi there, as I said in FB while ago, I'm pretty interested in contributing on this. What's the best way to begin with? @tado

**Comment by tado** [comment]
@kynd Thank you! How about to write in this issue which document translating now? When you finished translation please send pull request. I will marge into the repository.

**Comment by kynd** [comment]
sounds good. Are there any document that someone is already working on except for Installation_Up_4evr? Can I just pick whichever?

**Comment by tado** [comment]
@kynd There is no person working on ofBook Translation expect Installation_Up_4evr. Please pick up which you like. tpltnt said good start for the translation are 

- of_philosophy
- cplusplus_basics
- setup_and_project_structure
- intro_to_graphics
- OOPs!
- animation

Thank you for your support!!

**Comment by kynd** [comment]
@tado cool, I'll take a look. One more qq: is there any preference on writing style? If not I'll just try following http://openframeworks.cc/ja//
Thanks,

**Comment by tado** [comment]
@kynd Sorry, there is no style guide now. Please refer Jp documents on http://openframeworks.cc/ja/. Thanks!

**Comment by kynd** [comment]
ok, slowly working on it
https://github.com/kynd/ofBook/commit/c8e2f49d5e260466c42f2b975d59bd3959248282

**Comment by kynd** [comment]
@tado 
Just wanting to let you know it's going. Taking long time... 
https://github.com/kynd/ofBook/blob/master/chapters/cplusplus_basics/chapter.ja.md

**Comment by tado** [comment]
Hi kynd,

Oh, sorry. I overlooked your pull request. I will check and marge it.
Thanks!
​​

On Mon, May 23, 2016 at 6:08 AM, kynd <notifications@github.com> wrote:

> @tado <https://github.com/tado>
> Just wanting to let you know it's going. Taking long time...
>
> https://github.com/kynd/ofBook/blob/master/chapters/cplusplus_basics/chapter.ja.md
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openframeworks/ofBook/issues/185#issuecomment-220856510>
>



-- 
田所 淳 (たどころ あつし)

**Comment by kynd** [comment]
@tado no, it's still WIP:) I just wanted to let you know I'm still working on it since it's been a long time. Will let you know when it's done

**Comment by tado** [comment]
@kynd Oh ok. I understand... Thank you for your help!
